backend/app/Http/Controllers/SupportController.php

```
@@ -0,0 +1,41 @@
+{
+    public function store(Request $request): RedirectResponse
+    {
+        abort_if(config('settings.is_self_hosted'), 404);
+
+        $validated = $request->validate([
+            'message' => 'required|string|min:10|max:2000',
+        ]);
+
+        $user = auth()->user();
+
+        SupportMessage::create([
```

PHPStan (2.1.51)

[error] Call to an undefined static method App\Models\SupportMessage::create().


(staticMethod.notFound)